qml/TabLayout.qml

```diff
@@ -149,23 +210,37 @@ Item {
                         component_id: stack.stack_id
 
                         onUpdate_tab_name: {
-                            tabLayout.tab_model_[current_]["title"] = new_name
-
-                            // update model to set the visual change
-                            tab_list.model = tabLayout.tab_model_
-
-                            // update left panel information
-                            for (var i=0; i<stack_layout.count; i++)
+                            for (var i = 0; i<tabLayout.tab_model_.length; i++)
```

Comment:
Because previously we were considering that we were updating always the "current" focused tab, so we always rename the current one. This behaviour may change, so I though it was more conservative to check the given id to rename the associated tab instead, for escalability purposes and avoid possible future errors